relations.csv

```diff
@@ -4067,3 +4068,104 @@ HGNC,MIRLET7A3,isa,FPLX,MIRLET7A
 FPLX,MIRLET7F,isa,FPLX,MIRLET7
 HGNC,MIRLET7F1,isa,FPLX,MIRLET7F
 HGNC,MIRLET7F2,isa,FPLX,MIRLET7F
+FPLX,HTR1,isa,FPLX,5_hydroxytryptamine_receptors_G_protein_coupled
+FPLX,HTR2,isa,FPLX,5_hydroxytryptamine_receptors_G_protein_coupled
+FPLX,HTR5,isa,FPLX,5_hydroxytryptamine_receptors_G_protein_coupled
+HGNC,HTR4,isa,FPLX,5_hydroxytryptamine_receptors_G_protein_coupled
+HGNC,HTR6,isa,FPLX,5_hydroxytryptamine_receptors_G_protein_coupled
+HGNC,HTR7,isa,FPLX,5_hydroxytryptamine_receptors_G_protein_coupled
+FPLX,5_hydroxytryptamine_receptors_G_protein_coupled,isa,FPLX,5-HT
+HGNC,HTR1A,isa,FPLX,HTR1
+HGNC,HTR1B,isa,FPLX,HTR1
+HGNC,HTR1D,isa,FPLX,HTR1
+HGNC,HTR1E,isa,FPLX,HTR1
+HGNC,HTR2A,isa,FPLX,HTR2
+HGNC,HTR2B,isa,FPLX,HTR2
+HGNC,HTR2C,isa,FPLX,HTR2
+HGNC,HTR5A,isa,FPLX,HTR5
+HGNC,HTR5BP,isa,FPLX,HTR5

```

Comment:
`HTR5BP` is a pseudogene, we could remove it. That would leave `HTR5A` as the only member of `HTR5` meaning that we can also remove `HTR5` and link `HTR5A` directly to `HTR5`'s parent.